main.py

```diff
@@ -250,6 +250,9 @@ def ui_access_wrapper(self, *args, **kwargs):
     get_redirect_route(
         r'/contributionrightsdatahandler',
         admin.ContributionRightsDataHandler),
+    get_redirect_route(
+        r'/gettranslationcontributionstatshandler',
```

Comment:
```suggestion
        r'/translationcontributionstatshandler',
```

Maybe^?

(get is going to be the request method)

Reply:
Done.